package.json

```diff
@@ -44,6 +44,7 @@
     "Linters"
   ],
   "activationEvents": [
+    "onStartupFinished",
```

Comment:
good idea.

I tried manually and result is not that much good in my opinion. I do not have precise speed numbers but there is still delay between command execution and input prompt because of running process "Activation extensions..." even I put command registration on beginning of activate function :-(

the only way how to achieve immediate response is to have onStartup activation event